src/components/UI/ResourcesCard/ResourcesCard.tsx

```diff
@@ -34,19 +39,27 @@ export const ResourcesCard: FunctionComponent<ResourcesCardProps> = ({ details }
     watchLink,
     eventLink,
     eventSlides,
+    videoTimeStamps,
   } = details;
   const hasMedia = placeholderText || youtubeEmbedCode;
+  const [currentTimeStamp, setCurrentTimeStamp] = useState(0);
+
+  const convertSecondsToMinAndSec = (seconds: number): string => {
+    return `${~~(seconds / 60)}:${seconds % 60}m`;
```

Comment:
nice, din noe about double tilde ~~

Reply:
can be considered a utility, maybe can move to utils too i think

Reply:
ok